nomad/structs/structs.go

```diff
@@ -2174,6 +2174,10 @@ type Node struct {
 	// Raft Indexes
 	CreateIndex uint64
 	ModifyIndex uint64
+
+	// NodeMaxAllocs is only set if a NodeMaxAllocs value
+	// was set in the client config
+	NodeMaxAllocs int
```

Comment:
Bump up a few lines so its above the `*Index` fields. Right after `LastDrain`. Tradition :shrug: